x-pack/filebeat/input/awss3/sqs_input.go

```diff
@@ -226,19 +270,23 @@ func (w *sqsWorker) processMessage(ctx context.Context, msg types.Message) {
 		w.client.Publish(e)
 		publishCount++
```

Comment:
Really, this var should not be needed since the implementation already returns the sum of to-publish events in `result.eventCount`, but it seems that this is not implemented in testing due to the approach taken in mock testing.